.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐛 Bug report
+description: Create a report to help us reproduce and fix a bug
+labels: [bug]
+
+body:
+- type: textarea
+  attributes:
+    label: Description
+    description: >
+      Please provide a clear and concise description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to reproduce
```

Comment:
A lot of other repos have sections for "expected result" and "actual result" but I always find myself writing something like "I would expect this to work" and "it doesn't work" so I've never found those sections super useful. Hopefully users will clarify when things don't work as they would expect.

Reply:
➕